src/main/java/com/iota/iri/controllers/TransactionViewModel.java

```diff
@@ -118,6 +118,10 @@ public boolean update(final Tangle tangle, String item) throws Exception {
         getBranchTransactionHash();
         getBundleHash();
         getTagValue();
+        getObsoleteTagValue();
+        getAttachmentTimestamp();
```

Comment:
these are actual getters, they don't load values / parse fields.
`setAttachmentData()` should be used instead (also contains `tag`)